clio-server/src/main/scala/org/broadinstitute/clio/server/webservice/IndexWebService.scala

```diff
@@ -23,10 +25,12 @@ abstract class IndexWebService[CI <: ClioIndex](
 
   private[webservice] val postMetadata: Route = {
     pathPrefix("metadata") {
-      pathPrefixKey { key =>
-        post {
-          entity(as[indexService.clioIndex.MetadataType]) { metadata =>
-            complete(indexService.upsertMetadata(key, metadata))
+      parameter('force.as[Boolean] ? false) { force =>
```

Comment:
Yuck, it would take it as a `Symbol`... do you think it'd be worth sticking `"force"` in a constant in the `transfer-model` project, since it's scattered around a few places? We could (hopefully) still use it here with `Symbol(the-string-constant).as[Boolean]`